src/card_operations/card_fetch_encrypt_data.c

```diff
@@ -132,45 +187,12 @@ card_error_type_e card_fetch_encrypt_data(const uint8_t *wallet_id,
     }
 #endif
 
-    if (CARD_OPERATION_SUCCESS == card_data.error_type) {
+    if (card_data.error_type == CARD_OPERATION_SUCCESS) {
```

Comment:
i think this check is irrelevant, cause `card_fetch_wallet_name()` is taking `card_data` as value and its failure state is handled by its return value itself.